src/project.ts

```diff
@@ -332,6 +364,58 @@ class Project extends ServiceObject {
       }
     );
   }
+
+  getIamPolicy(): Promise<GetIamPolicyResponse>;
```

Comment:
Could you sort this method alphabetically within the method declarations? It makes it easier to navigate the file when jumping in for a quick fix / investigation down the road.